### DISCUSSION TITLE
Sarahboev's Assets

### DISCUSSION THREAD

**Discussion by zicklag** [fishfolk/jumpy#687] [Assets]
I'm creating this dicussion to track assets by `@Sarahboev` on Discord.

**Comment by zicklag** [comment]
## New Tileset & Revamped Decorations:

![coral rocks](https://user-images.githubusercontent.com/57463655/223883060-ecd563bd-8e3d-4aa7-b960-3a8477fe0528.png)
![decorations revamped](https://user-images.githubusercontent.com/57463655/223883063-0b0fee8b-856e-456f-aac8-70c95a526885.png)

**Comment by SirGiraffey** [comment]
![turtle](https://user-images.githubusercontent.com/57463655/224135680-dab7424e-c33c-45fb-830a-a4d346b7ecb7.png)
She also made a huge turtle as a background moving feature

**Reply by zicklag** [reply]
Ooh, that would be fun to insert between a couple of the parallax layers or something and really increase the perception of depth in the background.

**Reply by erlend-sh** [reply]
Yep, something like that was the idea. First place to experiment with it might be the credits screen though 🤔
